index.html

```diff
@@ -33,13 +33,18 @@
             mct.run();
         });
     </script>
+    <link rel="stylesheet" href="/platform/commonUI/general/res/css/startup-base.css">
+    <link rel="stylesheet" href="/platform/commonUI/general/res/css/openmct.css">
```

Comment:
Think there should not be leading slashes here; implies CSS paths are relative to the root, which is true in a dev environment, but not necessarily true once deployed.